apps/greencheck/object_storage.py

```diff
@@ -3,30 +3,53 @@
 Below is the wrapper around it for working with the objects.
 """
 import boto3
+import json
 from django.conf import settings
 
 
-def object_storage_bucket(bucket_name: str):
+# TODO: how to specify this return time. The following example throws an error as
+# this structure does not exist
+# def retrieve_object_storage() -> boto3.resources.factory.s3.ServiceResource:
```

Comment:
I can't seem to specify the return time like this.
This is what the documentation of boto3 and simply type() point out, but it is not accepted because the problem seems to be that s3 is not an attribute of factory.